Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.py

```diff
@@ -440,7 +464,7 @@ def prepare_args_for_custom_user_schema(args: dict[str, str]) -> dict[str, str]:
     if args.get('field_raw_json'):
         field_json = GSuiteClient.safe_load_non_strict_json(args['field_raw_json'])
     elif args.get('field_json_entry_id'):
-        field_json = safe_load_json(args['field_json_entry_id'])
+        field_json = return_file_from_entry_id(args['field_json_entry_id'])
```

Comment:
let's add this as a breaking change note
specify the updated commands and how this field is expected to be changed

Reply:
Done